### ISSUE TITLE
Possible rename of project?

### ISSUE THREAD

**Issue by drewhamlett** [grahammendick/navigation#306]
With react-navigation and react-native-navigation, this project is pretty much invisible on Google.  I'm guessing it's one of the reasons why it's not very well known.  I have trouble just finding docs for react-native-navigation instead of react-navigation.  A lot of people in react native community have never heard of this project, but in my testing it's much better than the other navigation solutions.  react-navigation is slow and we've kind of reached the limit on my current project.  Even with react-native-screens.  react-native-navigation is extremely buggy and simple things don't seem to work.  

I think with a rebrand/rename it can reach a lot more people.

**Comment by grahammendick** [comment]
Thanks for thinking of ways so get this project to a wider audience. I’ve spent many hours trying to crack this nut. I’ve written [many](https://medium.com/@graham.mendick/navigation-patterns-in-react-native-b000c6f8d474) [blog](https://medium.com/@graham.mendick/navigation-in-react-native-is-broken-ba5b1f245eb8) [posts](https://medium.com/@graham.mendick/three-apps-for-the-price-of-one-3f740f6918ed). I’ve applied to lots of conferences but been rejected every time. I’ve chatted to people on Twitter and reddit when they’ve expressed frustration with their existing navigation libraries. I don’t think it’s a problem with the name. I think that, unless you’re a React celebrity or your work has been endorsed by one, then there’s not much chance of breaking through.

It’s not all bad news, though. I’ll be speaking at a React Native meetup later this month. I’m hoping something good will come of that as long as I don’t let the nerves get the better of me!

**Comment by drewhamlett** [comment]
Oh nice! Good luck with the talk.  I don't think react-native-navigation is really blessed by anybody in the react native community.  Seems to only be react-navigation.  The biggest issue is Facebook doesn't care about navigation because they embed react-native in existing apps and I'm sure they have their own internal things for navigation.

Again though, this project is not discoverable anywhere.  I've come across the docs a few times over the past year or so, but I immediately don't even know what it is.  At least related to react native(where I think this project can have a big impact).  I can't get back to the docs from Google because I can't even search it.  The docs are not that good looking either, but I could help with that if you would like.  I think a little polish on docs can help projects go a long way in the JS community.  Just my 2 cents from being in this community for a while.  Another thing to help projects is put them in their own org on Github.  Developers like to see projects in orgs as opposed to a random project on some persons Github page.

You should try to open a PR for this page to get listed. https://facebook.github.io/react-native/docs/navigatorios.html .  

I know you're against naming but my initial thought was something like react-native-platform-navigation.  Other than that you would probably have to go for some kind of known or verb.  Some people just take a word and translate it to another language.  "navigation" or "navigate".

Thanks again!

**Comment by grahammendick** [comment]
Thanks for your thoughtful suggestions. 

When React Native announced they were rewriting their documentation I opened a [PR to get the Navigation router noticed](https://github.com/facebook/react-native-website/pull/1086) That raised the router’s profile a bit and I’m hoping it will get listed when the new docs launch.

Thanks for offering to polish the docs. Let me know what you’ve got planned and we’ll take it from there. Adding meta tags would help with discoverability.

The Navigation router isn’t only for React Native. It isn’t even only for React! The core `navigation` npm package works with any UI library. A while back I had folders for Angular, Knockout and CycleJS. I deleted them because it was stretching me too thin and I thought users would find it confusing. Focusing on React made it easier to document, too.

I don’t want to only target React Native. The Navigation router is just as important for the Web as it is for Native. There’s a lot of confusion about what to expect from a Web routing library. I created the [Router Challenge](https://github.com/grahammendick/router-challenge) to highlight the different use-cases and invited all library maintainers to take up the challenge. None of them did!
